ee/tabby-db/src/github_provider.rs

```diff
@@ -0,0 +1,57 @@
+use anyhow::{anyhow, Result};
+use sqlx::{query, query_as};
+
+use crate::DbConn;
+
+pub struct GithubProviderDAO {
+    pub name: String,
+    pub github_url: String,
+    pub application_id: String,
+    pub secret: String,
+}
+
+const GITHUB_PROVIDER_ROWID: i64 = 1;
```

Comment:
it can be multiple github provider. not a single row table.

Reply:
This is odd to me, then - the schema for a repository includes only a candidate ID, full URL, and name. Nothing here seems to indicate which provider it belongs to.